scripts/nginx-ubi/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+
+LABEL name="kabanero-index" \
+      vendor="Kabanero" \
+      version="1" \
+      release="1" \
+      summary="Nginx container to host Kabanero collections" \
```

Comment:
Can we call these Enterprise stacks rather than Kabanero collections?

Reply:
OK. Done.